composer.json

```diff
@@ -11,7 +11,7 @@
     ],
     "require": {
         "symfony/console": "~2.4",
-        "rhumsaa/uuid": "~2.7"
+        "ramsey/uuid": "~2.7"
```

Comment:
You changed to the newer version of the uuid package in the composer file but the use statement in the `ConvertCommand.php` file is unchanged. Is this correct?